packages/tables/demo/stories/TableStory.tsx

```diff
@@ -147,7 +151,11 @@ export const TableStory: Story<IArgs> = ({
                   </Cell>
                 )}
                 {Object.keys(row).map((column, columnIndex) => (
-                  <Cell key={`${rowIndex}${columnIndex}`} isTruncated={isTruncated}>
+                  <Cell
+                    key={`${rowIndex}${columnIndex}`}
+                    isTruncated={isTruncated}
+                    hidden={isHidden && rowIndex === 2 && columnIndex === 0}
```

Comment:
Maybe we can make the example the entire column of cells instead of a single cell?
```suggestion
                    hidden={isHidden && columnIndex === 0}
```

Reply:
Per JZ's suggestion, Hiding All the Cells™: [c33a847](https://github.com/zendeskgarden/react-components/pull/1514/commits/c33a847918a15f751b0ce425c5ae60c93ef3d771)